src/include/OpenImageIO/paramlist.h

```diff
@@ -117,6 +117,7 @@ class OIIO_API ParamValue {
         : m_name(p.m_name), m_nvalues(p.m_nvalues), m_interp(p.m_interp),
```

Comment:
Let's put it here, where the other initializers are (except for the ones that are part of unions, which is harder to do in the initializer list).